applications/nrf5340_audio/src/utils/channel_assignment.c

```diff
@@ -35,7 +35,7 @@ void channel_assignment_set(enum audio_channel channel)
 }
 #endif /* CONFIG_AUDIO_HEADSET_CHANNEL_RUNTIME */
 
-void channel_assignment_init(void)
```

Comment:
Not sure: maybe we could still make this API explicitly?

Reply:
I'm not sure what you mean by explicit in this context, can you elaborate?